contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -59,22 +50,18 @@ export const createAuthMiddleware = async (
     next: NextFunction,
   ) => {
     try {
-      const token =
-        getBearerTokenFromAuthorizationHeader(req.headers.authorization) ||
-        (req.cookies.token as string | undefined);
-      if (!token) {
-        res.status(401).send('Unauthorized');
-        return;
-      }
-      try {
-        req.user = await identity.authenticate(token);
-      } catch {
-        await appEnv.tokenManager.authenticate(token);
-      }
+      const token = getBearerTokenFromAuthorizationHeader(
+        req.headers.authorization,
+      );
```

Comment:
No need to grab the token here right? We'll get it from the `getIdentity` call anyway